### ISSUE TITLE
Include versions in your documentation

### ISSUE THREAD

**Issue by ckimmick** [MicrosoftDocs/OfficeDocs-Support#28]
Some directions hard to follow because the Excel version they apply to is not listed.  Please list the version so we know whether they apply to the version we have or know that they don't and we have to hunt around.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 62a88c46-43bb-92cf-c409-c8b3071b3865
* Version Independent ID: 19f5e98e-36a5-e2e9-39b1-5fc5ae349b50
* Content: [Troubleshoot Excel opening blank when double-click a file icon or file name - Office](https://docs.microsoft.com/en-us/office/troubleshoot/excel/excel-opens-blank#feedback)
* Content Source: [Office/Client/excel/excel-opens-blank.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Office/Client/excel/excel-opens-blank.md)
* Product: **office-perpetual-itpro**
* GitHub Login: @simonxjx
* Microsoft Alias: **v-six**

**Comment by simonxjx** [comment]
The applies to is added. https://docs.microsoft.com/en-us/office/troubleshoot/excel/excel-opens-blank
